frontend/pages/admin/IntegrationsPage/cards/MdmSettings/ScepPage/ScepPage.tsx

```diff
@@ -0,0 +1,379 @@
+import React, { useContext, useState, useEffect } from "react";
+import { useQuery } from "react-query";
+import { InjectedRouter } from "react-router";
+
+import PATHS from "router/paths";
+import configAPI from "services/entities/config";
+import { IConfig } from "interfaces/config";
+import { getErrorReason } from "interfaces/errors";
+import { NotificationContext } from "context/notification";
+import { AppContext } from "context/app";
+
+import MainContent from "components/MainContent/MainContent";
+import Button from "components/buttons/Button";
+import BackLink from "components/BackLink/BackLink";
+import CustomLink from "components/CustomLink";
+import Card from "components/Card";
+import validateUrl from "components/forms/validators/valid_url";
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import TooltipWrapper from "components/TooltipWrapper";
+import PremiumFeatureMessage from "components/PremiumFeatureMessage";
+import Spinner from "components/Spinner";
+import DataError from "components/DataError";
+import TurnOnMdmMessage from "components/TurnOnMdmMessage";
+
+import { SCEP_SERVER_TIP_CONTENT } from "../components/ScepSection/ScepSection";
+
+const baseClass = "scep-page";
+
+const BAD_SCEP_URL_ERROR = "Invalid SCEP URL. Please correct and try again.";
+const BAD_CREDENTIALS_ERROR =
+  "Invalid admin URL or credentials. Please correct and try again.";
+const CACHE_ERROR =
+  "The NDES password cache is full. Please increase the number of cached passwords in NDES and try again. By default, NDES caches 5 passwords and they expire 60 minutes after they are created.";
+const DEFAULT_ERROR =
+  "Something went wrong updating your SCEP server. Please try again.";
+
+interface IScepCertificateContentProps {
+  router: InjectedRouter;
+  onFormSubmit: (evt: React.MouseEvent<HTMLFormElement>) => Promise<void>;
+  formData: INdesFormData; // TODO
+  formErrors: INdesFormErrors;
+  onInputChange: ({ name, value }: IFormField) => void;
+  config: IConfig | null;
+  isPremiumTier: boolean;
+  isLoading: boolean;
+  isSaving: boolean;
+  saveButtonDisabled: boolean;
+  showDataError: boolean;
+}
+
+export const ScepCertificateContent = ({
+  router,
+  onFormSubmit,
+  formData,
+  formErrors,
+  onInputChange,
+  config,
+  isPremiumTier,
+  isLoading,
+  isSaving,
+  saveButtonDisabled,
+  showDataError,
+}: IScepCertificateContentProps) => {
+  if (!isPremiumTier) {
+    return <PremiumFeatureMessage />;
+  }
+
+  if (!config?.mdm.enabled_and_configured) {
+    return (
+      <TurnOnMdmMessage
+        router={router}
+        header="Turn on Apple MDM"
+        info="To help your end users connect to Wi-Fi, first turn on Apple MDM."
+      />
+    );
+  }
+
+  if (isLoading) {
+    return <Spinner />;
+  }
+
+  // TODO: error UI
```

Comment:
This TODO is left here intentionally and matches the `AppleBusinessManagerPage.tsx` and `VppPage.tsx` TODO